ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/Editor.tsx

```diff
@@ -122,13 +124,13 @@ function Editor(props: EditorProps, ref: any) {
 			mode: props.mode,
 			readOnly: props.readOnly,
 			autoCloseBrackets: props.autoMatchBraces,
-			inputStyle: 'textarea', // contenteditable loses cursor position on focus change, use textarea instead
+			inputStyle: 'contenteditable', // contenteditable loses cursor position on focus change, use textarea instead
```

Comment:
What do you make of the comment that was there - is the contenteditable no longer losing the cursor position on focus change?

Reply:
It looks like it's not, I tested this on my computer as well and I can't reproduce the original bug. I had run into this bug fairly early on in development and I suspect that enough changed with the structure of our interactions with codemirror that the bug stopped presenting itself. But I really can't be sure.

Reply:
It appears that bug doesn't exist anymore. I have confirmed this with @CalebJohn as well.